gcloud/logging/config_service_v2_api.py

```diff
@@ -14,64 +14,123 @@
 #
 # EDITING INSTRUCTIONS
 # This file was generated from the file
-# https://github.com/google/googleapis/blob/7710ead495227e80a0f06ceb66bdf3238d926f77/google/logging/v2/logging_config.proto,
+# https://github.com/google/googleapis/blob/master/google/logging/v2/logging_config.proto,
 # and updates to that file get reflected here through a refresh process.
 # For the short term, the refresh process will only be runnable by Google engineers.
-# Manual additions are allowed because the refresh process performs
-# a 3-way merge in order to preserve those manual additions. In order to not
-# break the refresh process, only certain types of modifications are
-# allowed.
 #
-# Allowed modifications:
-# 1. New methods (these should be added to the end of the class)
-#
-# Happy editing!
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+
+import os
+import platform
 
 from google.gax import api_callable
-from google.gax import api_utils
-from google.gax import page_descriptor
+from google.gax import config
+from google.gax.path_template import PathTemplate
+import google.gax
+import yaml
+
 from google.logging.v2 import logging_config_pb2
 
 
 class ConfigServiceV2Api(object):
+    """See src/api/google/logging/v2/logging.proto for documentation"""
 
-    # The default address of the logging service.
-    _SERVICE_ADDRESS = 'logging.googleapis.com'
+    _CODE_GEN_NAME_VERSION = 'gapic-0.1.0'
```

Comment:
This constant is used in a special tracking header
- this constant is deliberately
  * specified by the code generator and 
  * is not related to the package
- its purpose is to track the version of the code-generator that was used to generate the code, *not* the name and version of the package the generated code is used in
- note that there is application name and application version that can be specified the service constructor instances; it's there that the actual package version is intended to be used